en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+# Develop an API Proxy from Git Repository
+
+An API proxy acts as an intermediary between an existing API and Choreo, intercepting all requests made to the API. It also functions as a managed API, allowing you to apply essential API management features such as security policies and rate limiting.
+
+In this guide, you will:
+
+- Create an API proxy component to expose an existing API.
+- Deploy the API proxy.
+- Test the API proxy to verify its functionality.
+- Manage the API.
+- Consume the API.
+
+## Prerequisites
+
+- If you're signing in to the Choreo Console for the first time, create an organization:
+    1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in using your preferred method.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use.
+    4. Click **Create**.
+
+    This creates the organization and opens the **Project Home** page of the default project created for you.
+
+## Step 1: Create an API proxy
+
+To create an API proxy, you can either choose from a github repository, upload an OpenAPI specification or provide an OpenAPI specification URL. In this guide, you will choose the API definition from a Git source like Github repository.
+
+1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in. This opens the project home page.
+2. If you already have one or more components in your project, click **+ Create**. Otherwise, proceed to the next step.
+3. Click the **API Proxy** card. This opens the **Create an API Proxy** pane.
+4. You will see the three options to create a proxy component and here we select github option (selected by default).
+5. Specify the following values as component details:
+
+    !!! info
+        The **Component Name** field must be unique and cannot be changed after creation.System will generate a unique name for you. But you are free to change the generated name.
+        **Component Display Name** is a required field.
+
+    | **Field**       | **Value**                                  |
+    |-----------------|--------------------------------------------|
+    | **Component Display Name**| `Department Service`                    |
+    | **Component Name**        | `departmentService`                    |
+    | **Description**           | `This is a sample pet store`     |
```

Comment:
Need to update the description to align with the sample used here. Pet store is incorrect.